lib/net/common.js

```diff
@@ -227,3 +233,10 @@ exports.MAX_BLOCK_REQUEST = 50000 + 1000;
  */
 
 exports.MAX_TX_REQUEST = 10000;
+
+/**
+ ** Compact filter types.
+ */
+exports.FILTERS = {
+  REGULAR: 0
```

Comment:
They're calling this `BASIC` in BIP158, is that OK?
https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki#block-filters

Reply:
It's just a name.